kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js

```diff
@@ -135,13 +136,16 @@ define([
          * @returns {DataTable} datatable object
          */
         function renderTable(rows) {
-            const rowCount = rows.length;
+            // group jobs by their retry parents
+            const jobsByOriginalId = Jobs.getCurrentJobs(rows, jobsByRetryParent);
+            const rowCount = Object.keys(jobsByOriginalId).length;
+
             dataTable = $(container)
                 .find('table')
                 .dataTable({
-                    data: rows,
+                    data: Object.values(jobsByOriginalId),
                     rowId: (row) => {
-                        return 'job_' + row.job_id;
+                        return `job_${row.retry_parent || row.job_id}`;
```

Comment:
for rows where the job has been / can be retried, the `rowId` is set to the original `retry_parent` job ID.